docs/README.md

```diff
@@ -139,6 +139,7 @@ You can configure the string received by the `text` prop using the following var
 | Prop name           | Type      | Description                                                               | Default value |
 | ------------------- | --------- | ------------------------------------------------------------------------- | ------------- |
 | `linkToProductPage` | `Boolean` | Whether or not the `gift-name` block should be a link to the gift's product page. | `false`       |
+| `showProductName` | `Boolean` | Whether the `gift-name` block should show the product name instead the sku name | `false`       |
```

Comment:
We usually avoid boolean props to better allow the evolution of the framework and avoid creating multiple props that are related to the same thing.

What I suggest for you to do is to change this prop to accept two possible values: `'productName'` or `'skuName'`, defaulting to `'skuName'` of course.

Reply:
As requested, I have changed the type of prop, transforming it into an enumerate with the two suggested parameters and using skuName as the default one :)